### DISCUSSION TITLE
[Question] Run a PyQT5 app in a tab in flet

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#1995] [Q&A] [answered]
### Question

Good morning. I want to know if it's possible run a pyqt5 app in a tab in flet. I need to design a User interface when I can see level of tanks and other variables, but in flet I can't design that, because the images and their positioning are not as responsive as I expect, plus I need to place indicators and fields on top to update with the variables. Is there any way to do that? Thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
> I want to know if it's possible run a pyqt5 app in a tab in flet.

I'm afraid this is not possible.

**Reply by ingdesarr1** [reply]
Thanks
